src/99-appendices/08-coordinate-systems.md

```diff
@@ -151,6 +151,16 @@ Restricted keywords for the `<CoordSysType>CoordinateSystem` field in the
     [ACPC site](https://www.fieldtriptoolbox.org/faq/acpc/) on the FieldTrip
     toolbox wiki.
 
+-   `fsnative`: The subject-specific FreeSurfer T1w space. This corresponds to
+    the original scanner image after it is re-sampled and re-scaled
+    to an isotropic 1 mm 256x256x256 volume in RAS orientation. The origin MUST be the Anterior Commissure.
+
+-   `fsnativetkr`: The subject-specific FreeSurfer T1w space. The origin of
+    the coordinate system is at the center of isotropic 1 mm 256x256x256
+    volume (that is the voxel center is at slices (128, 128, 128)) in RAS
+    orientation. This corresponds to FreeSurfer's surface RAS coordinates (TKR coordinates).
+    The origin MUST be the Anterior Commissure. The ``IntendedFor`` MUST link to an MRI that represents the 256x256x256 volume in the FreeSurfer file format. The difference between this coordinate system and ``fsnative`` is that TKR coordinates are defined on the surface of the brain based on FreeSurfer.
+
```

Comment:
Freesurfer has several surfaces (pial, white, inflated... ), it is unclear which surface is meant